src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -0,0 +1,141 @@
+const getCurrencyFullName = require('../../../../common/currency').getCurrencyFullName;
+const Client              = require('../../../../base/client');
+const BinarySocket        = require('../../../../base/socket');
+const localize            = require('../../../../../_common/localize').localize;
+const Url                 = require('../../../../../_common/url');
+const isCryptocurrency    = require('../../../../../_common/base/currency_base').isCryptocurrency;
+
+const AccountClosure = (() => {
+    const form_selector = '#form_closure';
+    let $form,
+        $txt_other_reason,
+        $closure_loading,
+        $closure_container,
+        $success_msg,
+        $error_msg,
+        $crypto,
+        $fiat;
+
+    const onLoad = () => {
+        $txt_other_reason  = $('#other_reason');
+        $closure_loading   = $('#closure_loading');
+        $closure_container = $('#closure_container');
+        $success_msg       = $('#msg_main');
+        $error_msg         = $('#msg_form');
+        $crypto            = $('#crypto');
+        $fiat              = $('#change-fiat');
+        $form              = $(form_selector);
+
+        BinarySocket.wait('landing_company').then((response) => {
+            const company = response.landing_company.gaming_company || response.landing_company.financial_company;
+            const currencies = company.legal_allowed_currencies;
+            const current_currency = Client.get('currency');
+            const show_currencies = currencies.filter((val) => val !== current_currency);
+            if (show_currencies) {
+                show_currencies.forEach((currency) => {
+                    if (isCryptocurrency(currency)) {
+                        $crypto.find('ul').append(`<li>${getCurrencyFullName(currency)}</li>`);
+                    } else {
+                        $fiat.find('ul').append(`<li>${getCurrencyFullName(currency)}</li>`);
+                    }
+                });
+                if (!$crypto.find('li').length) $crypto.setVisibility(0);
+                if (!$fiat.find('li').length) $fiat.setVisibility(0);
+            }
+            const hasMT5 = response.landing_company.mt_gaming_company ||
```

Comment:
please use the function `MetaTrader.isEligible()`

Reply:
it doesn't tell you whether client has mt5 or not, just that they are allowed to have mt5 or not

Reply:
Yeah that's all we need. Thanks!